cmd/swag/main.go

```diff
@@ -29,9 +30,14 @@ const (
 	parseDepthFlag       = "parseDepth"
 	instanceNameFlag     = "instanceName"
 	overridesFileFlag    = "overridesFile"
+	quiet                = "quiet"
```

Comment:
I prefer `-q` flag in favor  of `-quiet`.

Reply:
> I prefer the `q` flag in favor of `quiet`. If this flag disables all info output, we may use `s` for silent.

I add a alias `q`, is that ok for you?
https://github.com/swaggo/swag/blob/quiet_flag/cmd/swag/main.go#L39

Reply:
ok